lib/fileutils.rb

```diff
@@ -1334,6 +1413,9 @@ def compare_file(a, b)
   # Returns +true+ if the contents of streams +a+ and +b+ are identical,
   # +false+ otherwise.
   #
+  # Arguments +a+ and +b+
+  # should be {interpretable as a path}[rdoc-ref:FileUtils@Path+Arguments].
```

Comment:
`a` and `b` are streams, not paths (they have to respond to the `read` method).

Reply:
Fixed.